infra/aws/terraform/registry.k8s.io/main.tf

```diff
@@ -69,8 +66,68 @@ module "us-east-2" {
     aws = aws.us-east-2
   }
 
-  region = "us-east-2"
   prefix = var.prefix
+
+  s3_replication_iam_role_arn = "arn:aws:iam::513428760722:role/registry.k8s.io_s3admin"
+
+  s3_replication_rules = [
+    {
+      id                               = "registry-k8s-io-us-east-2-to-registry-k8s-io-us-west-1"
+      status                           = "Enabled"
+      priority                         = 1
```

Comment:
Thanks for working on this @deobieta !

Just out of curiosity, are the rule priorities ordered in any meaningful way? I see in the [provider documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_replication_configuration#priority) that each rule has to have a unique value here, but are they set like this for any particular reason?

Reply:
No particular reason. Rule priority come in handy when object scope in rules overlap. In this case there's no actual filter for object scope (we want all the bucket objects),  thou we need to set it to use the S3 V2 API schema to set multiple destination buckets. Other way the API call would fail.